metadata-io/src/main/java/com/linkedin/metadata/entity/datastax/ConditionalWriteFailedException.java

```diff
@@ -0,0 +1,12 @@
+package com.linkedin.metadata.entity.datastax;
+
+public class ConditionalWriteFailedException extends RuntimeException {
```

Comment:
It does not look like this exception is used anywhere.

Reply:
Removed